apps/emqx_connector/i18n/emqx_connector_schema_lib.conf

```diff
@@ -68,13 +68,13 @@ emqx_connector_schema_lib {
 
     auto_reconnect {
```

Comment:
can be removed because it won't be generated in the doc.

Reply:
But if I removed it, I would get:

```
===< Generating: apps/emqx_dashboard/priv/www/static/bridge-api-en.json
{"init terminating in do_boot",{#{key=>{emqx_connector_schema_lib,"auto_reconnect"},reason=>no_desc_for_id,state=>#{file=>"apps/emqx_dashboard/priv/i18n.conf",tab=>#Ref<0.2744401323.1771175937.54031>}},[{hocon_schema,resolve_schema,2,[{file,"hocon_schema.erl"},{line,449}]},{emqx_dashboard_swagger,get_i18n,3,[{file,"emqx_dashboard_swagger.erl"},{line,455}]},{emqx_dashboard_swagger,trans_description,2,[{fil=ERROR REPORT==== 11-Jan-2023::21:43:13.860054 ===
Error in process <0.9.0> with exit value:
{#{key => {emqx_connector_schema_lib,"auto_reconnect"},
   reason => no_desc_for_id,
   state =>
       #{file => "apps/emqx_dashboard/priv/i18n.conf",
         tab => #Ref<0.2744401323.1771175937.54031>}},
 [{hocon_schema,resolve_schema,2,[{file,"hocon_schema.erl"},{line,449}]},
  {emqx_dashboard_swagger,get_i18n,3,
                          [{file,"emqx_dashboard_swagger.erl"},{line,455}]},
  {emqx_dashboard_swagger,trans_description,2,
                          [{file,"emqx_dashboard_swagger.erl"},{line,442}]},
  {emqx_dashboard_swagger,trans_desc,4,
                          [{file,"emqx_dashboard_swagger.erl"},{line,426}]},
  {emqx_dashboard_swagger,'-parse_object/3-fun-0-',4,
                          [{file,"emqx_dashboard_swagger.erl"},{line,821}]},
  {lists,foldl_1,3,[{file,"lists.erl"},{line,1355}]},
  {emqx_dashboard_swagger,parse_object,3,
e,"emqx_dashboard_swagger.erl"},{line,442}]},{em                          [{file,"emqx_dashboard_swagger.erl"},{line,813}]},
  {emqx_dashboard_swagger,components,4,
                          [{file,"emqx_dashboard_swagger.erl"},{line,566}]}]}
```